src/ui/statusbar.c

```diff
@@ -337,6 +337,9 @@ _status_bar_draw_extended_tabs(int pos)
         pos = _status_bar_draw_bracket(is_current, pos, "[");
 
         int status_attrs = theme_attrs(THEME_STATUS_ACTIVE);
```

Comment:
Maybe do:
int status_attrs;
and then use if/else.

Reply:
I guess then I should not change *from* that variant in the other place, as I did?

Reply:
I'll change any way you prefer. I just did it the way it's more readable for me, starting generic, going towards specific.

Reply:
It would just be nice not to assign a value twice if we can go around it.